src/ocm/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -29,6 +29,24 @@ const getManagementType = (clusterManagementType: boolean | undefined): string =
   return managementType;
 };
 
+const getDiskEncryptionType = (diskEncryption: string | undefined): string => {
```

Comment:
```suggestion
const getDiskEncryptionType = (diskEncryption: string | undefined): string => {
.............................................................................^
```
Is this any `string`? Can we constrain the types of strings this parameter can be assigned to?